buildSrc/src/main/kotlin/otel.errorprone-conventions.gradle.kts

```diff
@@ -20,6 +21,10 @@ tasks {
 
       excludedPaths.set(".*/build/generated/.*")
 
+      if (System.getenv("CI") == null) {
```

Comment:
I had thought it a necessary evil since the prevelance of printf debugging for us actually increases the chance of us accidentally merging it. But this is a great solution.